src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -583,6 +599,7 @@ def get_request_status(
         errored_gt,
         external_id,
     )
+    query = _sort_privacy_request_queryset(query, sort_due_date)
```

Comment:
A line logging `logger.info("Sorting requests by field: %s and direction: %s" % (sort_field, sort_direction))` would be good